### PR TITLE
Post Banner closes #68

### LINKED ISSUE #68: Post View Banner
## Description 
As a Po I want a user to be able to see a post on a separate page.

 **Describe the solution you'd like**
Create a Post View Banner 

 **Describe alternatives you've considered**
Does not apply

 **Additional context**
Does not apply

 **D.O.D.**
Profile Page banner should be according to design.

### PR DESCRIPTION
**Pull request by MiguelFieira**
closes #68 

 **What is it supposed to do**
If click on a post it should open a post view page with a banner according to design.

 **How can we test your branch**
- Go into branch ``68-post-view`` make sure to ``npm ci`` and click on a post on the homepage.
- Check of the banner is according to design

 **What browser have been checked**
- [X] Chrome
- [x] Firefox
- [x] Opera
- [ ] Internet Explorer
- [x] Edge
- [ ] Safari

### CODE REVIEW

**Comment by MiguelFieira** [comment]
**NOTES TO FIX:**
Edge is not correct!
image

Update: Changed the rgba so it looks better on edge!
image

File: src/components/postBanner/postBanner.scss [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+		min-width: 150px;
+		min-height: 150px;
+		width: 20vh;
+		height: 20vh;
+		max-width: 80vw;
+		max-height: 80vw;
+	}
+	@include box-shadow;
+
+	&Edit {
```
Comment (MiguelFieira) [inline src/components/postBanner/postBanner.scss:65]: Add Z-index!

File: src/stores/user.js [hunk 2]
```diff
@@ -3,8 +3,8 @@ import { types } from 'mobx-state-tree'
 const UserStore = types
   .model('UserStore', {
     loggedIn: types.optional(types.boolean, true),
-    picture: types.optional(types.string, 'src/static/dummy/user/profile.jpg'),
-    banner: types.optional(types.string, 'src/static/dummy/user/banner.jpg'),
+    picture: types.optional(types.string, '../src/static/dummy/user/profile.jpg'),
```
Comment (MiguelFieira) [inline src/stores/user.js:6]: @AjobK can you take a look at this if it's okay to stay like this!

File: src/components/postBanner/index.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    }
+
+    return (
+      <section className={styles.wrapper}>
+        <div className={styles.innerWrapper}>
+          {user.loggedIn && <span className={styles.pictureEdit}>
+            <Icon iconName={'Pen'} />
+          </span>}
+          <div className={styles.postUserinfo}>
+            <div className={styles.picture} style={{ backgroundImage: `url(${user.picture})` }}>
```
Comment (MiguelFieira) [inline src/components/postBanner/index.js:28]: Make sure to change the store when the back-end is ready!

File: src/components/postBanner/postBanner.scss (hunk 1 above)
Comment (MiguelFieira) [reply src/components/postBanner/postBanner.scss:65]: Fixed!

File: src/components/posts/index.js [hunk 4]
```diff
@@ -45,7 +45,7 @@ class Posts extends Component {
 
       article.style.backgroundColor = rgb
       article.classList.add(styles.contact)
-      let contactText =        document.createElement('div')
+      let contactText = document.createElement('div')
```
Comment (vQFire) [inline src/components/posts/index.js:48]: Can be constant since it does not change.

**Review by vQFire — COMMENTED** [review]
Is is more of a question that a comment. If it is answered I will approve!

**Comment by MiguelFieira** [comment]
Checked Chrome,Firefox,Opera,Edge!
IE11 was broken again
I don't own a Mac for Safari check!

File: src/components/posts/index.js (hunk 4 above)
Comment (MiguelFieira) [reply src/components/posts/index.js:48]: I'm instructed not to change anything further only linting sins it's not part of the issue i'm currently working on.
Please check again if you have any further comments @vQFire

**Review by vQFire — APPROVED** [review]
Looks Good!

**Review by AjobK — CHANGES_REQUESTED** [review]
Not responsive

**Review by Samvdg — APPROVED** [review]

File: src/stores/user.js (hunk 2 above)
Comment (AjobK) [reply src/stores/user.js:6]: No. This isn't working for me.

File: src/stores/user.js [hunk 5]
```diff
@@ -3,8 +3,8 @@ import { types } from 'mobx-state-tree'
...
+    banner: types.optional(types.string, '../src/static/dummy/user/banner.jpg'),
```
Comment (AjobK) [inline src/stores/user.js:7]: This isn't working either. Do not change this. This is simply dummy data

**Review by AjobK — CHANGES_REQUESTED** [review]
Also, the postBanner is not aligning with the header. The wrapper should have a maximum width of 1000px (wrapper-width). If smaller then it should go to 100% with a padding of $header-padding or something like this. This is 20px.

Do not use media queries, look at reference

**Comment by AjobK** [comment]
Also, don't use my letter size check. The font size is way too small compared to the role

**Pull request closed without merging**